### ISSUE TITLE
govc find with -guest.ipAddress argument returns more than one result

### ISSUE THREAD

**Issue by gokeefe** [vmware/govmomi#1089]
Given the command line `govc find . -type m -guest.ipAddress "10.128.40.46" -runtime.powerState poweredOn`, only one VM should be returned. However for some reason govc is returning 26 results, where only one has the specified guest.ipAddress.

```
$ govc find . -type m -guest.ipAddress "10.128.40.46" -runtime.powerState poweredOn | wc -l
26
```

The extra VM results don't have VMware Tools/open-vm-tools running so no guest ipAddress is being reported. Seems govc find is returning results where either the guest.ipAddress matches, or the guest.ipAddress is empty/null.

**Comment by dougm** [comment]
@gokeefe fixed by #1101 , can you confirm?

**Comment by gokeefe** [comment]
This seems to have fixed the issue I was experiencing, thank you!
